### DISCUSSION TITLE
complex business rules

### DISCUSSION THREAD

**Discussion by ahmed26399** [atrocore/atrocore#502] [Q&A]
Hey,
![MicrosoftTeams-image](https://user-images.githubusercontent.com/116101010/198337315-68202c6a-b0ac-4aeb-8131-00049d993232.png)

I'm trying to use complex business rules (for example, checking if the name is used in another record)
I  found this interface" formula"  but i m not sure if it would help me or not. 
How I can use formula ? Is there a documentation ?
Is there a solution to configure complex business rule ?

Thanks in advance

**Comment by atrolex** [comment]
We do not maintain formula functionality, it was inherited from EspoCRM. We do not know if and how this functionality is working.
What you try to implement will be possible with our updated Workflows module.
